test/Microsoft.AspNetCore.Razor.Test.MvcShim/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -16,5 +19,30 @@ public override void BeginContext(int position, int length, bool isLiteral)
         public override void EnsureRenderedBodyOrSections()
         {
         }
+
+        protected virtual IHtmlContent RenderBody()
+        {
+            return null;
+        }
+
+        public HtmlString RenderSection(string name)
+        {
+            return null;
+        }
+
+        public HtmlString RenderSection(string name, bool required)
+        {
+            return null;
+        }
+
+        public Task<HtmlString> RenderSectionAsync(string name)
+        {
+            return null;
+        }
+
+        public Task<HtmlString> RenderSectionAsync(string name, bool required)
+        {
+            return null;
+        }
```

Comment:
These aren't targeted by Razor code, but are used in the template's layout. I need them here so I can compile against the MvcShim